util/topsql/reporter/reporter.go

```diff
@@ -150,6 +165,7 @@ func (tsr *RemoteTopSQLReporter) RegisterPlan(planDigest []byte, normalizedPlan
 // Close implements TopSQLReporter.
 func (tsr *RemoteTopSQLReporter) Close() {
 	tsr.cancel()
+	tsr.sqlCPUCollector.Stop()
```

Comment:
What about other collectors? Do they need to be closed?

Reply:
There are no other collectors in `RemoteTopSQLReporter`.